styles/alternativeto/catppuccin.user.css

```
@@ -0,0 +1,481 @@
+        );
+        content: url("data:image/svg+xml,@{svg}");
+      }
+    }
+    // social networks
+    @socialnetworks: {
+      Facebook: @blue;
+      X: @text;
+    };
+    [class^="AppExternalLinks_socialLinksWrapper"] {
+      each(@socialnetworks, {
+        [title^="@{key} page"] > svg{
+          color: @value !important
+        }
+      });
+    }
+    // yello star (e.g. the level in a user profile)
+    svg[color="#e39d06"] {
+      color: @yellow !important;
+    }
+    // list like button hover
+    [class*="LikeButton_likeButton"]:hover [class*="LikeButton_heart"] {
+      color: @red;
+    }
+    // register popup header
+    .data-collector .header {
+      color: @overlay2;
+    }
+    // danger button (clear all filters)
+    [class*="Button_danger"] {
+      background: @red;
+      border-color: @red;
+      color: @crust;
+    }
+    // warning button
+    [class*="Button_warning"] {
+      background-color: @yellow;
+      border-color: @yellow;
+      color: @crust;
+    }
```

```suggestion
    // danger button (clear all filters)
    [class*="Button_danger"] {
      background: @red;
      border-color: @red;
      color: @crust;
    }
    // warning button
    [class*="Button_warning"] {
      background-color: @yellow;
      border-color: @yellow;
      color: @crust;
    }
    // success button
    [class*="Button_success"] {
      background-color: @green;
      border-color: @green;
      color: @crust;
    }
```

Going with the trend above we can add the success button as well :)